### PR TITLE
Updating Calendar

### PR DESCRIPTION
**Pull request by lnguy7**

### CODE REVIEW

File: src/content/Home/calendar.ts [hunk 1]
```diff
@@ -210,6 +210,28 @@ const events: Event[] = [
     initiatePointsCategory: "service",
     initiatePointsCount: 1,
   },
+  {
+    name: "Student Services Meeting",
+    date: new Date(2023, 8, 17),
```
Comment (alex-m-zhang) [inline src/content/Home/calendar.ts:215]: September, so 9, no? Not 8?

File: src/content/Home/calendar.ts [hunk 2]
```diff
@@ -210,6 +210,28 @@ const events: Event[] = [
...
+    virtual: false,
+    locationInfo: commonLocations[3013],
+    description:
+      "Preparation/Meeting for all things Student Services! (I.e. Review sessions, Worksheets, Videos, HKN wiki, etc)",
+    initiatePointsCategory: "service",
+    initiatePointsCount: 1,
+  },
+  {
+    name: "Student Services Meeting",
+    date: new Date(2023, 8, 24),
```
Comment (alex-m-zhang) [inline src/content/Home/calendar.ts:226]: September, so 9, no? Not 8?

**Review by alex-m-zhang — CHANGES_REQUESTED** [review]
Dates wrong?

File: src/content/Home/calendar.ts (hunk 2 above)
Comment (rzig) [reply src/content/Home/calendar.ts:226]: Months are 0-indexed

**Review by rzig — APPROVED** [review]

**Review by alex-m-zhang — APPROVED** [review]
LGTM

**Pull request merged**